src/lib/crypto/kimchi_backend/run_in_thread/dune

```diff
@@ -0,0 +1,8 @@
+(library
```

Comment:
where was this code copied from? Also, I think this is general enough that it should live in a `common/` or a `utility/` folder, not within `kimchi_backend/` or `crypto/`

Reply:
This was deleted from zexe_backend and I added it back